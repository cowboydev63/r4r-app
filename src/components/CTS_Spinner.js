import React from 'react';
import './CTS_Spinner.css';

const CTS_Spinner = () => (
    <div className="r4r-cts-spinner">
        <div className="spinkit spinner">
            <div className="dot1"></div>
            <div className="dot2"></div>
        </div>
    </div>
)

export default CTS_Spinner;