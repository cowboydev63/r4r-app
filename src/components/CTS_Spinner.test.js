import React from 'react';
import { shallow } from 'enzyme';
import CTS_Spinner from './CTS_Spinner';

describe('CTS_Spinner Component', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<CTS_Spinner/>);
        expect(wrapper).toMatchSnapshot();
    })
})