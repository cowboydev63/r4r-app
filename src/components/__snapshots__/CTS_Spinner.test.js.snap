// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTS_Spinner Component renders correctly 1`] = `
<div
  className="r4r-cts-spinner"
>
  <div
    className="spinkit spinner"
  >
    <div
      className="dot1"
    />
    <div
      className="dot2"
    />
  </div>
</div>
`;
